nearcore/src/migrations.rs

```diff
@@ -358,7 +360,12 @@ pub fn migrate_23_to_24(path: &String, near_config: &NearConfig) {
         let restored_receipts = serde_json::from_slice(&MAINNET_RESTORED_RECEIPTS)
             .expect("File with receipts restored after apply_chunks fix have to be correct").get(&0u64);
         let mut chain_store_update = ChainStoreUpdate::new(&mut chain_store);
-        chain_store_update.save_receipts(restored_receipts);
+        let mut store_update = chain_store_update.store().store_update();
+        for receipt in receipts.iter() {
+            let bytes = receipt.try_to_vec().expect("Borsh cannot fail");
+            store_update.update_refcount(ColReceipts, receipt.get_hash().as_ref(), &bytes, 1);
+        }
+        store_update.commit().expect("");
```

Comment:
```suggestion
        store_update.commit().unwrap();
```

Reply:
Also committing per value is a bit excessive. You can commit at the end since we know that there are not that many receipts ti begin with.

Reply:
But I don't commit per value, commit is outside the loop.